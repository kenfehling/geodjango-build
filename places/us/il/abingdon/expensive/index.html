
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Is Abingdon expensive?</title>
    <meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=yes,minimum-scale=0.25,maximum-scale=5">
    <link rel="stylesheet" type="text/css" href="/static/style.css">
    <link rel="shortcut icon" href="/static/favicon/favicon.ico">
    <link rel="icon" sizes="16x16 32x32 64x64" href="/static/favicon/favicon.ico">
    <link rel="icon" type="image/png" sizes="196x196" href="/static/favicon/favicon-192.png">
    <link rel="icon" type="image/png" sizes="160x160" href="/static/favicon/favicon-160.png">
    <link rel="icon" type="image/png" sizes="96x96" href="/static/favicon/favicon-96.png">
    <link rel="icon" type="image/png" sizes="64x64" href="/static/favicon/favicon-64.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/static/favicon/favicon-32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/static/favicon/favicon-16.png">
    <link rel="apple-touch-icon" href="/static/favicon/favicon-57.png">
    <link rel="apple-touch-icon" sizes="114x114" href="/static/favicon/favicon-114.png">
    <link rel="apple-touch-icon" sizes="72x72" href="/static/favicon/favicon-72.png">
    <link rel="apple-touch-icon" sizes="144x144" href="/static/favicon/favicon-144.png">
    <link rel="apple-touch-icon" sizes="60x60" href="/static/favicon/favicon-60.png">
    <link rel="apple-touch-icon" sizes="120x120" href="/static/favicon/favicon-120.png">
    <link rel="apple-touch-icon" sizes="76x76" href="/static/favicon/favicon-76.png">
    <link rel="apple-touch-icon" sizes="152x152" href="/static/favicon/favicon-152.png">
    <link rel="apple-touch-icon" sizes="180x180" href="//static/favicon/favicon-180.png">
    <meta name="msapplication-TileColor" content="#FFFFFF">
    <meta name="msapplication-TileImage" content="//static/favicon/favicon-144.png">
    <meta name="msapplication-config" content="/static/favicon/browserconfig.xml">
    
    <meta name="keywords" content="Abingdon, Illinois, IL, expensive, housing, cost, price, statistics, location, places, real estate">
    <meta name="description" content="In comparison to other places in this area, Abingdon is a bit less expensive, and it is significantly less expensive compared to the rest of the nation.">
    <meta name="og:description" content="In comparison to other places in this area, Abingdon is a bit less expensive, and it is significantly less expensive compared to the rest of the nation.">
    
        <script type="application/ld+json">{"@context": "https://schema.org", "@type": "Article", "author": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "publisher": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "datePublished": "2021-08-04T05:06:00.701976", "dateModified": "2021-08-04T05:06:00.701976", "image": "https://storage.googleapis.com/statistipedia/img/maps/png/il/abingdon.png", "headline": "Is Abingdon expensive?", "mainEntity": {"@type": "City", "name": "Abingdon", "sameAs": "https://www.wikidata.org/wiki/Q579672", "containedInPlace": {"@type": "State", "name": "Abingdon", "sameAs": "https://www.wikidata.org/wiki/Q1204"}}}</script>
    

    <meta name="og:locale" content="en_US" />
</head>

<body>
<header>
    <div class="inner-header">
        <a class="site-title" href="/">
            Statistipedia
        </a>
    </div>
</header>

<main>
    <article>
        <h1>Is Abingdon expensive?</h1>
        
    

<table class="infobox">
    <tr>
        <th colspan="2" class="title">Abingdon, IL</th>
    </tr>
    <tr>
        <td colspan="2" class="place-image">
            <picture>
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/webp/il/abingdon.webp" type="image/webp" />
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/png/il/abingdon.png" type="image/png" />
    <img src="https://storage.googleapis.com/statistipedia/img/maps/png/il/abingdon.png" loading="lazy" alt="Map of Abingdon" />
</picture>
        </td>
    </tr>
    <tbody class="main-info">
        <tr>
            <th>State</th>
            <td>
                <a href="/places/us/il">Illinois</a>
            </td>
        </tr>
        <tr>
            <th>Population</th>
            <td>3,509</td>
        </tr>
    </tbody>
    <tr>
        <th colspan="2" class="header">Links</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                
                
                    <li>
                        <a href="https://en.wikipedia.org/wiki/Abingdon,_Illinois" target="_blank" rel="nofollow noopener">
                            Wikipedia article
                        </a>
                    </li>
                
                <li>
                    <a href="https://data.census.gov/cedsci/profile?g=1600000US1700113" target="_blank" rel="nofollow noopener">
                        Census profile
                    </a>
                </li>
            </ul>
        </td>
    </tr>
    <tr>
        <th colspan="2" class="header">Maps</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                <li><a target="_blank" rel="nofollow noopener" href="https://www.openstreetmap.org/search?query=Abingdon%2C%20IL">OpenStreetMaps</a></li>
                <li><a target="_blank" rel="nofollow noopener" href="https://www.google.com/maps/place/Abingdon,+IL">Google Maps</a></li>
                <li><a target="_blank" rel="nofollow noopener" href="https://www.bing.com/maps?q=Abingdon,+IL">Bing Maps</a></li>
                <li><a target="_blank" rel="nofollow noopener" href="https://www.mapquest.com/search/result?query=Abingdon,+IL">MapQuest</a></li>
            </ul>
        </td>
    </tr>
</table>
    
        <p class="lede">In comparison to other places in this area, <a href="/places/us/il/abingdon">Abingdon</a> is a bit less expensive, and it is significantly less expensive compared to the rest of the nation. Compared to the city of <a href="/places/us/il/peoria/expensive?">Peoria</a>, it is significantly less expensive.</p>
        <h2>Home prices</h2>
        <p>The median home in Abingdon costs $61,700. Compared to its nearby areas this is about $14,640 (19%) lower. It is roughly $155,800 (71%) lower compared to the country as a whole, and  is about $64,000 (50%) lower compared to the city of Peoria.</p>
        <div class="viz-wrapper">
            
<div class="figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <div class="viz">
                
                    <div class="country bar">
                        <div style="width: 100.0% ">United States</div>
                    </div>
                
                    <div class="city bar">
                        <div style="width: 57.793103448275865% ">Peoria</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 48.7816091954023% ">Knoxville</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 47.724137931034484% ">East Galesburg</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 34.89655172413793% ">Galesburg</div>
                    </div>
                
                    <div class="location bar">
                        <div style="width: 28.367816091954023% ">Abingdon</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 23.54022988505747% ">Avon</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 20.551724137931036% ">London Mills</div>
                    </div>
                
            </div>
        </div>
        <figcaption>Comparison of median Abingdon home price to nearby areas</figcaption>
    </figure>
</div>
            
<div class="distro figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <iframe src="/static/plot/index.html?ax[]=0&ax[]=%3C50k&ax[]=50k&ax[]=100k&ax[]=150k&ax[]=200k&ax[]=300k&ax[]=500k&ax[]=1m&c[]=0&c[]=69&c[]=120&c[]=133&c[]=140&c[]=196&c[]=193&c[]=114&c[]=34&n[]=0&n[]=203&n[]=490&n[]=180&n[]=72&n[]=30&n[]=16&n[]=9&n[]=0&l[]=0&l[]=375&l[]=487&l[]=121&l[]=16&l[]=0&l[]=0&l[]=0&l[]=0&a[]=0&a[]=126&a[]=254&a[]=218&a[]=156&a[]=138&a[]=81&a[]=24&a[]=4" loading="lazy"></iframe>
        </div>
        <figcaption>Distribution of home prices for Abingdon and nearby areas</figcaption>
    </figure>
</div>

        </div>
        <h2>Rents</h2>
        <p>Abingdon's median rent  is $547 a month. Compared to its surrounding areas it is roughly $114 (17%) lower. Compared to the United States as a whole it is roughly $515 (48%) lower, and it is roughly $259 (32%) lower in comparison to the city of Peoria.</p>
        <div class="viz-wrapper">
            
<div class="figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <div class="viz">
                
                    <div class="country bar">
                        <div style="width: 100.0% ">United States</div>
                    </div>
                
                    <div class="city bar">
                        <div style="width: 75.89453860640302% ">Peoria</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 72.97551789077212% ">London Mills</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 63.088512241054616% ">East Galesburg</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 61.111111111111114% ">Knoxville</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 59.98116760828625% ">Galesburg</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 53.954802259887% ">Avon</div>
                    </div>
                
                    <div class="location bar">
                        <div style="width: 51.50659133709981% ">Abingdon</div>
                    </div>
                
            </div>
        </div>
        <figcaption>Comparison of median Abingdon rent to nearby areas</figcaption>
    </figure>
</div>
            
<div class="distro figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <iframe src="/static/plot/index.html?ax[]=0&ax[]=%3C500&ax[]=500&ax[]=1000&ax[]=1500&ax[]=2000&ax[]=2500&ax[]=3000&c[]=0&c[]=94&c[]=362&c[]=300&c[]=140&c[]=56&c[]=24&c[]=24&n[]=0&n[]=278&n[]=620&n[]=87&n[]=11&n[]=0&n[]=0&n[]=4&l[]=0&l[]=352&l[]=624&l[]=24&l[]=0&l[]=0&l[]=0&l[]=0&a[]=0&a[]=156&a[]=580&a[]=206&a[]=37&a[]=10&a[]=3&a[]=9" loading="lazy"></iframe>
        </div>
        <figcaption>Distribution of rents for Abingdon and nearby areas</figcaption>
    </figure>
</div>

        </div>
        <h2>Nearby places</h2>
        <h3>Knoxville</h3><p><a href="/places/us/il/knoxville/expensive?">Knoxville</a> is about 8 miles northeast from Abingdon. Its real estate prices are significantly more expensive. Its median home price is $106,100, which works out to $44,400 (41%) higher than Abingdon.</p><h3>London Mills</h3><p><a href="/places/us/il/london-mills/expensive?">London Mills</a> is roughly 8 miles southeast of Abingdon. Housing prices there are a lot less expensive. The median home value there is $44,700, which comes out to $17,000 (38%) lower than Abingdon.</p><h3>Avon</h3><p><a href="/places/us/il/avon/expensive?">Avon</a> is approximately 9 miles south from Abingdon. The housing prices there are a bit less expensive. Its median home cost is $51,200, which works out to $10,500 (20%) lower than Abingdon.</p><h3>Galesburg</h3><p><a href="/places/us/il/galesburg/expensive?">Galesburg</a> is approximately 9 miles north of Abingdon. Its real estate prices are somewhat more expensive. Its median home value is $75,900, which works out to $14,200 (18%) higher than Abingdon.</p><h3>East Galesburg</h3><p><a href="/places/us/il/east-galesburg/expensive?">East Galesburg</a> is approximately 10 miles northeast from Abingdon. Its housing prices are a lot more expensive. The median home value there is $103,800, which works out to $42,100 (40%) higher than Abingdon.</p>
    

        
        <ul>
            
        </ul>
        <nav class="categories">
            Categories:
            &nbsp;<a href="/places/us">United States</a>
            &nbsp;<span class="divider">|</span>
            &nbsp;<a href="/places/us/il">Illinois</a>
        </nav>
    </article>
</main>

<footer>
    <div class="inner-footer">
        <a class="site-title" href="/">
            Statistipedia
        </a>
        <div>

        </div>
        <ul>
            <li>© 2021 Statistipedia</li>
            <li><a href="/about">About</a></li>
            <li><a href="/privacy">Privacy</a></li>
        </ul>
    </div>
</footer>
</body>
</html>