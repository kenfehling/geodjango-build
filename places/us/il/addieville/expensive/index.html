<!DOCTYPE html>
<html lang="en">
<head>
    
<meta charset="UTF-8">
<title>Is Addieville expensive?</title>
<meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=yes,minimum-scale=0.25,maximum-scale=5">
<link rel="stylesheet" type="text/css" href="/static/style.css">
<link rel="shortcut icon" href="/static/favicon.ico"/>

</head>
<body>
    <header>
    <div class="inner-header">
        <a class="site-title" href="/">
            Statistipedia
        </a>
    </div>
</header>
    <main>
        <article>
            <h1>Is Addieville expensive?</h1>
            

<table class="infobox">
    <tr>
        <th colspan="2" class="title">Addieville, IL</th>
    </tr>
    <tr>
        <td colspan="2" class="place-image">
            <picture>
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/webp/il/addieville.webp" type="image/webp" />
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/png/il/addieville.png" type="image/png" />
    <img src="https://storage.googleapis.com/statistipedia/img/maps/png/il/addieville.png" alt="Map of Addieville" />
</picture>
        </td>
    </tr>
    <tbody class="main-info">
        <tr>
            <th>State</th>
            <td>
                <a href="/places/us/il">Illinois</a>
            </td>
        </tr>
        <tr>
            <th>Population</th>
            <td>373</td>
        </tr>
    </tbody>
    <tr>
        <th colspan="2" class="header">Links</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                
                
                <li>
                    <a href="https://data.census.gov/cedsci/profile?g=1600000US1700230" target="_blank">
                        Census profile
                    </a>
                </li>
            </ul>
        </td>
    </tr>
    <tr>
        <th colspan="2" class="header">Maps</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                <li><a target="_blank" href="https://www.openstreetmap.org/search?query=Addieville%2C%20IL">OpenStreetMaps</a></li>
                <li><a target="_blank" href="https://www.google.com/maps/place/Addieville,+IL">Google Maps</a></li>
                <li><a target="_blank" href="https://www.bing.com/maps?q=Addieville,+IL">Bing Maps</a></li>
                <li><a target="_blank" href="https://www.mapquest.com/search/result?query=Addieville,+IL">MapQuest</a></li>
            </ul>
        </td>
    </tr>
</table>
            
                <p class="lede">The value of the average home in <a href="/places/us/il/addieville">Addieville</a> is $98,600.</p>
                <p class="lede">Compared to nearby areas, Addieville is a bit less expensive, and it's substantially less expensive compared to the US as a whole. Compared to <a href="/places/us/mo/st-louis/expensive?">St. Louis</a>, it's somewhat less expensive.</p>
                <div class="figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <div class="viz">
                
                    <div class="country bar"
                         style="width: 100.0%">
                        US average
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 54.46666666666667%">
                        Damiansville
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 51.333333333333336%">
                        Bartelso
                    </div>
                
                    <div class="city bar"
                         style="width: 46.233333333333334%">
                        St. Louis
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 42.9%">
                        Nashville
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 38.13333333333333%">
                        Venedy
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 37.7%">
                        Okawville
                    </div>
                
                    <div class="location bar"
                         style="width: 32.86666666666667%">
                        Addieville
                    </div>
                
            </div>
        </div>
        <figcaption>Comparison of average Addieville home price to nearby areas</figcaption>
    </figure>
</div>
                <h2>Nearby places</h2><h3>Damiansville</h3><p><a href="/places/us/il/damiansville/expensive?">Damiansville</a> is 10 miles northwest of Addieville. Housing prices there are substantially more expensive. The average home value there is $163,400, which is $64,800 (65%) higher than Addieville.</p><h3>Bartelso</h3><p><a href="/places/us/il/bartelso/expensive?">Bartelso</a> is 10 miles north of Addieville. The real estate prices there are significantly more expensive. The average home cost there is $154,000, which works out to $55,400 (56%) higher than Addieville.</p><h3>Nashville</h3><p>The city of <a href="/places/us/il/nashville/expensive?">Nashville</a> is 5 miles southeast of Addieville. The housing prices there are a bit more expensive. Its average home value is $128,700, which comes out to $30,100 (30%) higher than Addieville.</p><h3>Venedy</h3><p><a href="/places/us/il/venedy/expensive?">Venedy</a> is 7 miles west of Addieville. The real estate prices there are somewhat more expensive. Its average home cost is $114,400, which comes out to $15,800 (16%) higher than Addieville.</p><h3>Okawville</h3><p><a href="/places/us/il/okawville/expensive?">Okawville</a> is 4 miles northwest of Addieville. The real estate prices there are a little more expensive. Its average home cost is $113,100, which is $14,500 (14%) higher than Addieville.</p>
                <h2>See also</h2>
<ul><li><a href="/places/us/il/st-libory/expensive?">St. Libory</a></li>
<li><a href="/places/us/il/germantown/expensive?">Germantown</a></li>
<li><a href="/places/us/il/richview/expensive?">Richview</a></li>
<li><a href="/places/us/il/fayetteville/expensive?">Fayetteville</a></li>
<li><a href="/places/us/il/new-baden/expensive?">New Baden</a></li>
<li><a href="/places/us/il/albers/expensive?">Albers</a></li>
<li><a href="/places/us/il/hoyleton/expensive?">Hoyleton</a></li></ul>
            
            <nav class="categories">
                Categories:
                &nbsp;<a href="/places/us">United States</a>
                &nbsp;<span class="divider">|</span>
                &nbsp;<a href="/places/us/il">Illinois</a>
            </nav>
        </article>
    </main>
    <footer>
    <div class="inner-footer">
        <a class="site-title" href="/">
            Statistipedia
        </a>
        <div>

        </div>
        <ul>
            <li>Â© 2021 Statistipedia</li>
            <li><a href="/about">About</a></li>
            <li><a href="/privacy">Privacy</a></li>
        </ul>
    </div>
</footer>
</body>
</html>