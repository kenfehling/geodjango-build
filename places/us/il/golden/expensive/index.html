
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Is Golden expensive?</title>
    <meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=yes,minimum-scale=0.25,maximum-scale=5">
    <link rel="stylesheet" type="text/css" href="/static/style.css">
    <link rel="shortcut icon" href="/static/favicon/favicon.ico">
    <link rel="icon" sizes="16x16 32x32 64x64" href="/static/favicon/favicon.ico">
    <link rel="icon" type="image/png" sizes="196x196" href="/static/favicon/favicon-192.png">
    <link rel="icon" type="image/png" sizes="160x160" href="/static/favicon/favicon-160.png">
    <link rel="icon" type="image/png" sizes="96x96" href="/static/favicon/favicon-96.png">
    <link rel="icon" type="image/png" sizes="64x64" href="/static/favicon/favicon-64.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/static/favicon/favicon-32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/static/favicon/favicon-16.png">
    <link rel="apple-touch-icon" href="/static/favicon/favicon-57.png">
    <link rel="apple-touch-icon" sizes="114x114" href="/static/favicon/favicon-114.png">
    <link rel="apple-touch-icon" sizes="72x72" href="/static/favicon/favicon-72.png">
    <link rel="apple-touch-icon" sizes="144x144" href="/static/favicon/favicon-144.png">
    <link rel="apple-touch-icon" sizes="60x60" href="/static/favicon/favicon-60.png">
    <link rel="apple-touch-icon" sizes="120x120" href="/static/favicon/favicon-120.png">
    <link rel="apple-touch-icon" sizes="76x76" href="/static/favicon/favicon-76.png">
    <link rel="apple-touch-icon" sizes="152x152" href="/static/favicon/favicon-152.png">
    <link rel="apple-touch-icon" sizes="180x180" href="//static/favicon/favicon-180.png">
    <meta name="msapplication-TileColor" content="#FFFFFF">
    <meta name="msapplication-TileImage" content="//static/favicon/favicon-144.png">
    <meta name="msapplication-config" content="/static/favicon/browserconfig.xml">
    
    <meta name="keywords" content="Golden, Illinois, IL, expensive, housing, cost, price, statistics, location, places, real estate">
    <meta name="description" content="In comparison to the surrounding areas, Golden is a bit more expensive, and it&#x27;s substantially less expensive compared to the entire country.">
    <meta name="og:description" content="In comparison to the surrounding areas, Golden is a bit more expensive, and it&#x27;s substantially less expensive compared to the entire country.">
    
        <script type="application/ld+json">{"@context": "https://schema.org", "@type": "Article", "author": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "publisher": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "datePublished": "2021-08-04T05:00:31.229037", "dateModified": "2021-08-04T05:00:31.229037", "image": "https://storage.googleapis.com/statistipedia/img/maps/png/il/golden.png", "headline": "Is Golden expensive?", "mainEntity": {"@type": "City", "name": "Golden", "containedInPlace": {"@type": "State", "name": "Golden", "sameAs": "https://www.wikidata.org/wiki/Q1204"}}}</script>
    

    <meta name="og:locale" content="en_US" />
</head>

<body>
<header>
    <div class="inner-header">
        <a class="site-title" href="/">
            Statistipedia
        </a>
    </div>
</header>

<main>
    <article>
        <h1>Is Golden expensive?</h1>
        
    

<table class="infobox">
    <tr>
        <th colspan="2" class="title">Golden, IL</th>
    </tr>
    <tr>
        <td colspan="2" class="place-image">
            <picture>
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/webp/il/golden.webp" type="image/webp" />
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/png/il/golden.png" type="image/png" />
    <img src="https://storage.googleapis.com/statistipedia/img/maps/png/il/golden.png" loading="lazy" alt="Map of Golden" />
</picture>
        </td>
    </tr>
    <tbody class="main-info">
        <tr>
            <th>State</th>
            <td>
                <a href="/places/us/il">Illinois</a>
            </td>
        </tr>
        <tr>
            <th>Population</th>
            <td>597</td>
        </tr>
    </tbody>
    <tr>
        <th colspan="2" class="header">Links</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                
                
                <li>
                    <a href="https://data.census.gov/cedsci/profile?g=1600000US1730159" target="_blank" rel="nofollow noopener">
                        Census profile
                    </a>
                </li>
            </ul>
        </td>
    </tr>
    <tr>
        <th colspan="2" class="header">Maps</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                <li><a target="_blank" rel="nofollow noopener" href="https://www.openstreetmap.org/search?query=Golden%2C%20IL">OpenStreetMaps</a></li>
                <li><a target="_blank" rel="nofollow noopener" href="https://www.google.com/maps/place/Golden,+IL">Google Maps</a></li>
                <li><a target="_blank" rel="nofollow noopener" href="https://www.bing.com/maps?q=Golden,+IL">Bing Maps</a></li>
                <li><a target="_blank" rel="nofollow noopener" href="https://www.mapquest.com/search/result?query=Golden,+IL">MapQuest</a></li>
            </ul>
        </td>
    </tr>
</table>
    
        <p class="lede">In comparison to the surrounding areas, <a href="/places/us/il/golden">Golden</a> is a bit more expensive, and it's substantially less expensive compared to the entire country. Compared to the city of <a href="/places/us/il/springfield/expensive?">Springfield</a>, it is significantly less expensive.</p>
        <h2>Home prices</h2>
        <p>The median home cost in Golden is $82,700. This is about $9,340 (12%) higher in comparison to its nearby areas. Compared to the country as a whole it is roughly $134,800 (61%) lower, and it is roughly $50,700 (38%) lower in comparison to the city of Springfield.</p>
        <div class="viz-wrapper">
            
<div class="figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <div class="viz">
                
                    <div class="country bar">
                        <div style="width: 100.0% ">United States</div>
                    </div>
                
                    <div class="city bar">
                        <div style="width: 61.333333333333336% ">Springfield</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 49.37931034482759% ">Camp Point</div>
                    </div>
                
                    <div class="location bar">
                        <div style="width: 38.02298850574713% ">Golden</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 35.86206896551724% ">Coatsburg</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 30.942528735632184% ">Clayton</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 26.850574712643677% ">Augusta</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 25.60919540229885% ">Bowen</div>
                    </div>
                
            </div>
        </div>
        <figcaption>Comparison of median Golden home price to nearby areas</figcaption>
    </figure>
</div>
            
<div class="distro figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <iframe src="/static/plot/index.html?ax[]=0&ax[]=%3C50k&ax[]=50k&ax[]=100k&ax[]=150k&ax[]=200k&ax[]=300k&ax[]=500k&ax[]=1m&c[]=0&c[]=69&c[]=120&c[]=133&c[]=140&c[]=196&c[]=193&c[]=114&c[]=34&n[]=0&n[]=246&n[]=407&n[]=242&n[]=61&n[]=31&n[]=3&n[]=4&n[]=6&l[]=0&l[]=167&l[]=528&l[]=213&l[]=79&l[]=0&l[]=14&l[]=0&l[]=0&a[]=0&a[]=100&a[]=249&a[]=234&a[]=157&a[]=157&a[]=79&a[]=23&a[]=2" loading="lazy"></iframe>
        </div>
        <figcaption>Distribution of home prices for Golden and nearby areas</figcaption>
    </figure>
</div>

        </div>
        <h2>Rents</h2>
        <p>Golden's median rent  is $467 a month. This is approximately $132 (22%) lower in comparison to other places in this area. Compared to the rest of the nation it is approximately $595 (56%) lower, and it is roughly $338 (41%) lower in comparison to the city of Springfield.</p>
        <div class="viz-wrapper">
            
<div class="figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <div class="viz">
                
                    <div class="country bar">
                        <div style="width: 100.0% ">United States</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 87.09981167608287% ">Coatsburg</div>
                    </div>
                
                    <div class="city bar">
                        <div style="width: 75.80037664783427% ">Springfield</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 53.2015065913371% ">Camp Point</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 52.54237288135593% ">Bowen</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 51.2241054613936% ">Augusta</div>
                    </div>
                
                    <div class="location bar">
                        <div style="width: 43.973634651600754% ">Golden</div>
                    </div>
                
                    <div class="neighbor bar">
                        <div style="width: 37.94726930320151% ">Clayton</div>
                    </div>
                
            </div>
        </div>
        <figcaption>Comparison of median Golden rent to nearby areas</figcaption>
    </figure>
</div>
            
<div class="distro figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <iframe src="/static/plot/index.html?ax[]=0&ax[]=%3C500&ax[]=500&ax[]=1000&ax[]=1500&ax[]=2000&ax[]=2500&ax[]=3000&c[]=0&c[]=94&c[]=362&c[]=300&c[]=140&c[]=56&c[]=24&c[]=24&n[]=0&n[]=585&n[]=341&n[]=52&n[]=22&n[]=0&n[]=0&n[]=0&l[]=0&l[]=611&l[]=389&l[]=0&l[]=0&l[]=0&l[]=0&l[]=0&a[]=0&a[]=124&a[]=623&a[]=195&a[]=34&a[]=11&a[]=11&a[]=2" loading="lazy"></iframe>
        </div>
        <figcaption>Distribution of rents for Golden and nearby areas</figcaption>
    </figure>
</div>

        </div>
        <h2>Nearby places</h2>
        <h3>Camp Point</h3><p><a href="/places/us/il/camp-point/expensive?">Camp Point</a> is roughly 5 miles southwest of Golden. Its real estate prices are somewhat more expensive. Its median home cost is $107,400, which works out to $24,700 (22%) higher than Golden.</p><h3>Clayton</h3><p><a href="/places/us/il/clayton/expensive?">Clayton</a> is approximately 6 miles southeast of Golden. Its real estate prices are a little less expensive. Its median home cost is $67,300, which is $15,400 (22%) lower than Golden.</p><h3>Coatsburg</h3><p><a href="/places/us/il/coatsburg/expensive?">Coatsburg</a> is roughly 8 miles southwest from Golden. Housing prices there are somewhat less expensive. The median home value there is $78,000, which is $4,700 (6%) lower than Golden.</p><h3>Bowen</h3><p><a href="/places/us/il/bowen/expensive?">Bowen</a> is approximately 8 miles north of Golden. Its real estate prices are significantly less expensive. The median home cost there is $55,700, which is $27,000 (48%) lower than Golden.</p><h3>Augusta</h3><p><a href="/places/us/il/augusta/expensive?">Augusta</a> is roughly 8 miles north from Golden. Housing prices there are substantially less expensive. The median home value there is $58,400, which comes out to $24,300 (41%) lower than Golden.</p>
    

        
        <ul>
            
        </ul>
        <nav class="categories">
            Categories:
            &nbsp;<a href="/places/us">United States</a>
            &nbsp;<span class="divider">|</span>
            &nbsp;<a href="/places/us/il">Illinois</a>
        </nav>
    </article>
</main>

<footer>
    <div class="inner-footer">
        <a class="site-title" href="/">
            Statistipedia
        </a>
        <div>

        </div>
        <ul>
            <li>© 2021 Statistipedia</li>
            <li><a href="/about">About</a></li>
            <li><a href="/privacy">Privacy</a></li>
        </ul>
    </div>
</footer>
</body>
</html>